docs/website/root/manual/getting-started/run-signer-node.md

```diff
@@ -113,6 +113,32 @@ Note that this guide works on a Linux machine only.
 
 :::
 
+:::info
+
+To verify that your node version is supported, compare the version of your Cardano node with the minimum supported versions listed in the `cardano-min-versions.json` file located at the root of the repository.
+
+First, check the version of your Cardano node by running the following command:
+
+```bash
+cardano-node --version
+```
+
+Then, refer to the minimum supported versions listed in the `cardano-min-versions.json` file, which can be found [here](https://github.com/input-output-hk/mithril/blob/main/cardano-min-versions.json).
```

Comment:
```suggestion
Then, refer to the minimum supported versions listed in the [`cardano-min-versions.json`](https://github.com/input-output-hk/mithril/blob/main/cardano-min-versions.json) file, which can be found [here.
```

Reply:
Done ✅. I also removed the end ", which can be found here." to avoid a repetition.